scripts/tizen/4.0/packaging/caffe2.spec

```diff
@@ -0,0 +1,180 @@
+# If you have to disable a specific package due to a package absence
+# from https://git.tizen.org/cgit/, append -Dxxx_xxx=OFF option before executing cmake.
+#  -DUSE_BENCHMARK=OFF \
+#  -DCMAKE_INSTALL_PREFIX:PATH=/home/abuild/caffe2_deploy ..
+cmake \
+    -DCMAKE_VERBOSE_MAKEFILE=1 \
+    -DBUILD_TEST=OFF \
+    -DUSE_GFLAGS=ON  \
+    -DUSE_GLOG=ON \
+    -DUSE_NNPACK=OFF \
+    -DRUN_HAVE_STD_REGEX=0 \
+    -DRUN_HAVE_POSIX_REGEX=0 \
+    -DHAVE_GNU_POSIX_REGEX=0 \
+    -DUSE_MPI=OFF \
+    -DUSE_OPENMP=ON \
+    -DUSE_ROCKSDB=OFF \
+    -DUSE_LEVELDB=ON \
+    -DUSE_LMDB=ON \
+    -DBUILD_PYTHON=ON \
+    -DUSE_GLOO=OFF \
+    -DUSE_OPENCV=ON \
+    -DUSE_CUDA=OFF \
+%ifarch armv7l
+    -DCAFFE2_CPU_FLAGS="-mfpu=neon -mfloat-abi=soft" \
+%endif
+%ifarch aarch64
+    -DCAFFE2_CPU_FLAGS="-mtune=cortex-a53 -mabi=lp64" \
+%endif
+    .. \
+    || exit 1
+echo -e "Building Caffe2"
+make %{?_smp_mflags} || exit 1
+
+# verify a share file that is generated for Caffe2/CPU
+size ./caffe2/libCaffe2_CPU.so 
+size --format=sysv ./caffe2/libCaffe2_CPU.so 
+popd
+
+%install
+pushd build
+# create folders
+mkdir -p %{buildroot}%{_bindir}
+mkdir -p %{buildroot}%{_libdir}
+mkdir -p %{buildroot}%{_includedir}/caffe2/core/
+mkdir -p %{buildroot}%{_includedir}/caffe2/proto/
+mkdir -p %{buildroot}%{_includedir}/caffe/proto/
+mkdir -p %{buildroot}%{python_sitelib}/caffe2/
+
+# binary tools
+install -m 0755 -p ./caffe2/binaries/convert_caffe_image_db %{buildroot}%{_bindir}
+install -m 0755 -p ./caffe2/binaries/convert_db %{buildroot}%{_bindir}
+install -m 0755 -p ./caffe2/binaries/db_throughput %{buildroot}%{_bindir}
+install -m 0755 -p ./caffe2/binaries/make_cifar_db %{buildroot}%{_bindir}
+install -m 0755 -p ./caffe2/binaries/make_mnist_db %{buildroot}%{_bindir}
+install -m 0755 -p ./caffe2/binaries/predictor_verifier %{buildroot}%{_bindir}
+install -m 0755 -p ./caffe2/binaries/print_registered_core_operators %{buildroot}%{_bindir}
+install -m 0755 -p ./caffe2/binaries/run_plan %{buildroot}%{_bindir}
+install -m 0755 -p ./caffe2/binaries/speed_benchmark %{buildroot}%{_bindir}
+install -m 0755 -p ./caffe2/binaries/split_db %{buildroot}%{_bindir}
+
+# for CPU
+# TODO: support another architecture such as GPU, NPU.
+install -m 0644 -p ./caffe2/libCaffe2_CPU.so %{buildroot}%{_libdir}
+
+# for -devel
+install -m 0644 -p ./caffe2/core/*.h  %{buildroot}%{_includedir}/caffe2/core/
+install -m 0644 -p ./caffe2/proto/*.h  %{buildroot}%{_includedir}/caffe2/proto/
+
+# to convert caffe to caffe2 models
+install -m 0644 -p ./caffe/proto/caffe.pb.h  %{buildroot}%{_includedir}/caffe/proto/
+
+# install python packages
+cp -rf ./caffe2/python/* %{buildroot}%{python_sitelib}/caffe2/
+
+popd
+
+%post -p /sbin/ldconfig
+
+%postun -p /sbin/ldconfig
+
+%files
+%defattr(-,root,root)
+%manifest %{name}.manifest
+%doc ./LICENSE
+%doc ./PATENTS
+%{_bindir}/*
+%{_libdir}/libCaffe2_CPU.so
+
+%files devel
+%{_includedir}/caffe2/*
+%{_includedir}/caffe/proto/*
```

Comment:
We'd better .pc file later. (not so required for the initial commit though)

Reply:
Good point. If we need to append package config  file (e.g., *.pc) in the future, I will update existing build procedure with `pkg-config`.